### PR TITLE
New Recipe: raylib v4.0.0

### PR DESCRIPTION
**Pull request by chengchingwen**

### CODE REVIEW

File: R/raylib/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="alsa_jll", uuid="45378030-f8ea-5b20-a7c7-1a9d95efb90e"))
+    Dependency(PackageSpec(name="Mesa_jll", uuid="78dcde23-ec64-5e07-a917-6fe22bbc0f45"))
+    Dependency(PackageSpec(name="Xorg_libX11_jll", uuid="4f6342f7-b3d2-589e-9d20-edeb45f2b2bc"))
+    Dependency(PackageSpec(name="Xorg_libXrandr_jll", uuid="ec84b674-ba8e-5d96-8ba1-2a689ba10484"))
+    Dependency(PackageSpec(name="Xorg_libXi_jll", uuid="a51aa0fd-4e3c-5386-b890-e753decda492"))
+    Dependency(PackageSpec(name="Xorg_libXcursor_jll", uuid="935fb764-8cf2-53bf-bb30-45bb1f8bf724"))
+    Dependency(PackageSpec(name="Xorg_libXinerama_jll", uuid="d1454406-59df-5ea1-beac-c340f2130bc3"))
+    Dependency(PackageSpec(name="Xorg_xorgproto_jll", uuid="c4d99508-4286-5418-9131-c86396af500b"))
```
Comment (giordano) [inline R/raylib/build_tarballs.jl:50]:
```suggestion
    BuildDependency(PackageSpec(name="Xorg_xorgproto_jll", uuid="c4d99508-4286-5418-9131-c86396af500b"))
```

**Review by giordano — COMMENTED** [review]
Duplicate of  #3652, it'd be good to incorporate the differences

**Comment by chengchingwen** [comment]
Oh, I didn't notice there was another PR. What differences should be incorporate?

**Comment by chengchingwen** [comment]
@giordano Is this what you mean?

**Comment by giordano** [comment]
Allright, I added a couple of platforms more.  Windows is more messy.  Thanks!

**Comment by chengchingwen** [comment]
Thanks!

**Pull request merged**